shared/src/main/scala/com/raquo/domtypes/generic/defs/tags/SvgTags.scala

```diff
@@ -0,0 +1,733 @@
+package com.raquo.domtypes.generic.defs.tags
+
+import com.raquo.domtypes.generic.builders.TagBuilder
+
+trait SvgTags[T[_ <: DomElement], DomElement, SvgElement <: DomElement,
+    SvgCircle <: SvgElement,
+    SvgClipPath <: SvgElement,
+    SvgDefs <: SvgElement,
+    SvgDesc <: SvgElement,
+    SvgEllipse <: SvgElement,
+    SvgFEBlend <: SvgElement,
+    SvgFEColorMatrix <: SvgElement,
+    SvgComponentTransferFunction <: SvgElement,
+    SvgFEComposite <: SvgElement,
+    SvgFEConvolveMatrix <: SvgElement,
+    SvgFEDiffuseLighting <: SvgElement,
+    SvgFEDisplacementMap <: SvgElement,
+    SvgFEDistantLight <: SvgElement,
+    SvgFEFlood <: SvgElement,
+    SvgFEFuncA <: SvgElement,
+    SvgFEFuncB <: SvgElement,
+    SvgFEFuncG <: SvgElement,
+    SvgFEFuncR <: SvgElement,
+    SvgFEGaussianBlur <: SvgElement,
+    SvgFEImage <: SvgElement,
+    SvgFEMerge <: SvgElement,
+    SvgFEMergeNode <: SvgElement,
+    SvgFEMorphology <: SvgElement,
+    SvgFEOffset <: SvgElement,
+    SvgFEPointLight <: SvgElement,
+    SvgFESpecularLighting <: SvgElement,
+    SvgFESpotLight <: SvgElement,
+    SvgFETile <: SvgElement,
+    SvgFETurbulence <: SvgElement,
+    SvgFilter <: SvgElement,
+    SvgG <: SvgElement,
+    SvgImage <: SvgElement,
+    SvgLine <: SvgElement,
+    SvgLinearGradient <: SvgElement,
+    SvgMarker <: SvgElement,
+    SvgMask <: SvgElement,
+    SvgMetadata <: SvgElement,
+    SvgPath <: SvgElement,
+    SvgPattern <: SvgElement,
+    SvgPolygon <: SvgElement,
+    SvgPolyline <: SvgElement,
+    SvgRadialGradient <: SvgElement,
+    SvgRectElement <: SvgElement,
+    Svg <: SvgElement,
+    SvgStop <: SvgElement,
+    SvgSVG <: SvgElement,
+    SvgSwitch <: SvgElement,
+    SvgSymbol <: SvgElement,
+    SvgText <: SvgElement,
+    SvgTextPath <: SvgElement,
+    SvgTSpan <: SvgElement,
+    SvgUse <: SvgElement,
+    SvgView <: SvgElement
+  ] {
+  this: TagBuilder[T, SvgElement] =>
+
+
+  /**
+   * The altGlyph element allows sophisticated selection of the glyphs used to
+   * render its child character data.
+   *
+   * MDN
+   */
+  lazy val altGlyph:T[SvgElement] = tag("altGlyph")
```

Comment:
Formatting: Pls add a space between `:` and `T` in all these lazy vals to be consistent with other traits like this